### PR TITLE
Better descriptions for `minValue` settings

### PR DESCRIPTION
**Pull request by Felanbird**
Noticed the sub-par description of `clueMinValue` while testing #98, it took me too long to realize it wasn't talking about minValue for discord embeds.

The `minLootValue` description was better, but I went ahead and updated it anyway.

### CODE REVIEW

File: src/main/java/dinkplugin/DinkPluginConfig.java [hunk 1]
```diff
@@ -612,7 +612,7 @@ default boolean clueShowItems() {
     @ConfigItem(
         keyName = "clueMinValue",
         name = "Min Value",
-        description = "The minimum value of the items to be shown",
+        description = "The minimum value of the items for a notification to be sent",
```
Comment (pajlada) [inline src/main/java/dinkplugin/DinkPluginConfig.java:615]: Same question as #98 - should we specify that this checks the value of all items combined?

File: src/main/java/dinkplugin/DinkPluginConfig.java [hunk 2]
```diff
@@ -414,7 +414,7 @@ default boolean lootIcons() {
     @ConfigItem(
         keyName = "minLootValue",
         name = "Min Loot value",
-        description = "Minimum value of the loot to notify",
+        description = "Minimum value of the loot for a notification to be sent",
```
Comment (pajlada) [inline src/main/java/dinkplugin/DinkPluginConfig.java:417]: Clarify that this is the minimum value of an item stack?

**Review by pajlada — CHANGES_REQUESTED** [review]

File: src/main/java/dinkplugin/DinkPluginConfig.java (hunk 2 above)
Comment (Felanbird) [reply src/main/java/dinkplugin/DinkPluginConfig.java:417]:
```suggestion
        description = "The minimum value of an item for a notification to be sent",
```
Would the user recognize `Dragon bolts (unf) x 100` as being a single item, if it was displayed this way?

Comment (pajlada) [reply src/main/java/dinkplugin/DinkPluginConfig.java:417]: I would imagine this is clear enough yeah

**Review by iProdigy — APPROVED** [review]

**Review by pajlada — APPROVED** [review]

**Pull request merged**